stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/ChatDomainImplCreateChannelTest.kt

```diff
@@ -205,25 +201,6 @@ internal class ChatDomainImplCreateChannelTest {
             }
         }
 
-    @Test
-    fun `Given failed network response When create distinct channel Should return failed response And do not insert any channel to DB`() =
```

Comment:
Why this test has been removed?
Have we change our logic?
Can't we configure this tests to follow the same approach?
If the new logic produce new results, can't we verify the new results are as expected?

Reply:
We completely removed this method from offline module. And test was a kind of dummy. Check if network request failed we didn't do anything and returned failed result.